server/index.js

```diff
@@ -6,6 +6,9 @@ const path = require('path')
 const cookieParser = require('cookie-parser')
 const productsRouts = require('./routes/products')
 const cartRouts = require('./routes/carts')
+const userRouts = require('./routes/users')
```

Comment:
@AStashevskaya More correct variant `userRoutes` instead of `userRouts`, same for `productsRouts` and `cartRouts`